metadata-models/src/main/pegasus/com/linkedin/identity/CorpUserHomePageSettings.pdl

```diff
@@ -9,6 +9,10 @@ record CorpUserHomePageSettings {
   /**
    * The page template that will be rendered in the UI by default for this user
    */
+  @Relationship = {
+    "name": "HasPersonalPageTemplate",
+    "entityTypes": [ "dataHubPageTemplate" ]
+  }
   pageTemplate: optional Urn
```

Comment:
adding relationship here in case we need to connect users to selected templates later on (which we will)

Reply:
nice!